### PR TITLE
Fix lint issues and warnings

### PR DESCRIPTION
**Pull request by nb-ohad**
- Adjust spaces
- Start errors with a lowercase letter
- multi-line for long compound  statements

### CODE REVIEW

**Review by bindrad — CHANGES_REQUESTED** [review]
space should be added in kubebuilder markers in the offering_controller file

https://github.com/red-hat-storage/managed-fusion-agent/pull/21/files#diff-15ecb6b6ffa838b8c6cbc94f2a8d7ecd88e65ed59e0f0e818cca80b0af43be07R42-R43

File: api/v1alpha1/groupversion_info.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the misf v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=misf.ibm.com
+// +kubebuilder:object:generate=true
+// +groupName=misf.ibm.com
```
Comment (leelavg) [inline api/v1alpha1/groupversion_info.go:19]: - afaik, build markers shouldn't have space b/n marker & comment

Comment (nb-ohad) [reply api/v1alpha1/groupversion_info.go:19]: Maybe, but every time i save a file in my editor the default golang beautifier kicks in and adds a space

Comment (ezio-auditore) [reply api/v1alpha1/groupversion_info.go:19]: Same for me. lets have this change

**Pull request closed without merging**